ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-presubmits.yaml

```diff
@@ -62,7 +62,7 @@ presubmits:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-assisted-service-master-e2e-metal-assisted
     rerun_command: /test e2e-metal-assisted
-    run_if_changed: ^(cmd/.*|deploy/.*|hack/.*|internal/.*|pkg/.*|tools/.*|Dockerfile\..*|Makefile|go\.mod|go\.sum|swagger.yaml)$
+    run_if_changed: ^(cmd/.*|data/.*|deploy/.*|hack/.*|internal/.*|pkg/.*|tools/.*|Dockerfile\..*|Makefile|go\.mod|go\.sum|swagger.yaml)$
```

Comment:
anything needs to be changed on ``ocm-2.3``?

Reply:
no. the multi versions is cloud related

Reply:
but it has also ``default_hw_requirements.json``

Reply:
those variables are handled via env variables. Those are just the defaults that are used in the cloud :)

Reply:
We shouldn't backport data values

Reply:
ok, if it's not relevant to the operator then it's ok